data/dbus/snapd.conf

```diff
@@ -0,0 +1,3 @@
+<busconfig>
+ <servicedir>/var/lib/snapd/dbus-1/services/</servicedir>
```

Comment:
Can we please use "dbus/services"? That "dbus-1" convention always feels awkward to me, and we don't have many reasons to reproduce it here I think. The compatibility issues and their solution will be the same as for every other service we hold data for.